spec/discoverGranules/DiscoverGranules.input.template.json

```diff
@@ -0,0 +1,55 @@
+{
+  "cumulus_meta": {
+    "message_source": "sfn",
+    "state_machine": "",
+    "execution_name": ""
+  },
+  "meta": {
+    "workflow_name": "DiscoverGranules",
+    "stack": "",
+    "buckets": {
+      "internal": "",
+      "private": ""
+    },
+    "provider": {
+      "id": "http_provider",
+      "host": "http://ec2-54-159-27-56.compute-1.amazonaws.com:3030",
+      "protocol": "http"
+    },
+    "collection": {
+      "provider_path": "granules/fake_granules",
```

Comment:
Is this using the test-data package?

Reply:
http://ec2-54-159-27-56.compute-1.amazonaws.com:3030/granules/fake_granules
This is the temporary endpoint but yes it set up similarly to the test-data package.